### ISSUE TITLE
remove explicit svg inline styles

### ISSUE THREAD

**Issue by shrugs** [urbit/sigil-js#37]
the generated svg element has an inline width and height style, making responsive overrides ugly or difficult.

the inline styles are applied here: https://github.com/urbit/sigil-js/blob/master/lib/src/index.js#L164

and result in an SVG that does not scale to its container

explicit width/height: 
<img width="510" alt="image" src="https://user-images.githubusercontent.com/1535001/61741296-688e0f80-ad45-11e9-99a4-9ecefdce4b39.png">

unset width and height (default behavior is to expand to parent size):

<img width="514" alt="image" src="https://user-images.githubusercontent.com/1535001/61741327-79d71c00-ad45-11e9-85ec-fa855d451252.png">

to be clear: responsive height is preferred. only the viewbox is necessary, since that tells the browser how an svg should be scaled.

if a consumer wants a sigil to be an explicit and constant size they can easily do that with

```jsx
<div style={{width: size, height: size}}>
  {sigil({size, renderer: reactRenderer, ...})}
</div>
```

wheras to override these styles I must have a global style rule like

```scss
svg {
  width: inherit !important;
  height: inherit !important;
}
```

which is ugly and will definitely come back to bite a programmer in the future.

solving this issue will also solve: #30

**Comment by g-a-v-i-n** [comment]
I can see why this is important to resolve. A defined width and height is often preferred because the library changes the stroke width with the symbol size. For most sizes this isn't that big of an issue, but when the SVG goes below 64px or so, it becomes important to adjust strokeWidth in this way so that the sigil is visually clear. There is also a `margin` parameter in the config that can be set if you need precise sizing.

I need some extra time to look into a solution that works here, so I might not be able to get to this for a week or so.

**Comment by shrugs** [comment]
the caller should still set `size`, so that the stroke width and viewbox is set correctly, if that's what you mean. the idea is that the caller would set an 'expected' size the sigil would display at, but then be free to display it at more or less depending on the responsive parent.

**Comment by shrugs** [comment]
as in my proposed change is to just delete lines 164 and 165

**Comment by g-a-v-i-n** [comment]
@shrugs wanna check out #39 ?

**Comment by g-a-v-i-n** [comment]
resolved in #39 and published in 1.3.5
